vochain/keykeeper/keykeeper.go

```diff
@@ -507,3 +481,27 @@ func (k *KeyKeeper) signAndSendTx(tx *models.AdminTx) error {
 	}
 	return nil
 }
+
+// OnProcessKeys does nothing
+func (k *KeyKeeper) OnProcessKeys(pid []byte, pub string, txindex int32) {
+	// do nothing
+}
+
+// OnRevealKeys does nothing
+func (k *KeyKeeper) OnRevealKeys(pid []byte, priv string, txindex int32) {
+	// do nothing
```

Comment:
Delete this message and unify with other methods.

`func (k *KeyKeeper) OnRevealKeys(pid []byte, priv string, txindex int32) {}`